### PR TITLE
Update Datenanfragen for Hacktoberfest 2020

### LINKED ISSUE #702: Update Datenanfragen for Hacktoberfest 2020
## Issue with an existing swag opportunity

- [x] I've checked that this isn't a duplicate issue.

Q. Which opportunity?  
A. : Datenanfragen.de e. V.

This year, the Datenanfragen.de e. V. is doing another round of swag for the Hacktoberfest (see [here](https://www.datarequests.org/blog/hacktoberfest-2020/)).

### PR DESCRIPTION
**Pull request by zner0L**
Closes #702

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Update the Datenanfragen.de e. V. entry with information for Hacktoberfest 2020.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -129,10 +129,10 @@
     "name": "Datenanfragen.de e. V.",
     "difficulty": "medium",
     "description": "One merged PR gets at least 3 stickers (limited to 100 sets) and a chance at a limited edition t-shirt.",
```
Comment (aslafy-z) [inline data.json:131]: `at least 3 stickers`, is this still valid?
On their website, they call it a `free sticker set`

File: data.json [hunk 2]
```diff
@@ -129,10 +129,10 @@
...
-    "reference": "https://www.datarequests.org/blog/hacktoberfest-2019/",
-    "image": "https://www.datarequests.org/blog/hacktoberfest-2019/hacktoberfest-2019.png",
+    "reference": "https://www.datarequests.org/blog/hacktoberfest-2020/",
+    "image": "https://www.datarequests.org/blog/hacktoberfest-2020/hacktoberfest-2020_hu28b1cba020bf43bc24568088e4124bec_241852_881x0_resize_q75_box.jpg",
     "dateAdded": "2019-10-23T00:39:55.000Z",
```
Comment (aslafy-z) [inline data.json:134]: Please bump the date so it goes first in the list.

File: data.json [hunk 3]
```diff
@@ -129,10 +129,10 @@
     "name": "Datenanfragen.de e. V.",
```
Comment (aslafy-z) [inline data.json:129]: Shouldn't we rename this entry to datarequests.org ?

**Comment by vikaspotluri123** [comment]
@zner0L just FYI - issues won't get linked if you mention them in the PR title; If you put that text in the PR description, it will, though :smile:

File: data.json (hunk 1 above)
Comment (zner0L) [reply data.json:131]: Yup, still valid. I am a maintainer of datenanfragen, so I can definitely say that :)

File: data.json (hunk 2 above)
Comment (zner0L) [reply data.json:134]:
```suggestion
    "dateAdded": "2020-10-01T19:00:00.000Z",
```

**Review by vikaspotluri123 — APPROVED** [review]

**Pull request merged**